static/js/src/advantage/subscribe/checkout/components/Taxes/Taxes.tsx

```diff
@@ -60,7 +59,7 @@ const Taxes = ({ setError }: TaxesProps) => {
   const onSaveClick = () => {
     setIsEditing(false);
     setFieldTouched("isTaxSaved", false);
-    if (isGuest || !window.accountId) {
+    if (!window.accountId) {
```

Comment:
One question for tax section - when it's logged in as a fresh user and there's no country saved, cancel button shouldn't be shown, right? Compared to the current checkout, it's different. Is it intentional? 
![image](https://github.com/canonical/ubuntu.com/assets/57550290/6ad56a73-e2ed-42d7-b016-1fe8dfa8368c)